syncapi/storage/postgres/output_room_events_topology_table.go

```diff
@@ -50,14 +50,14 @@ const insertEventInTopologySQL = "" +
 const selectEventIDsInRangeASCSQL = "" +
 	"SELECT event_id FROM syncapi_output_room_events_topology" +
 	" WHERE room_id = $1 AND (" +
-	"(topological_position > $2 AND topological_position < $3) OR" +
+	"(topological_position > $2 AND topological_position <= $3) OR" +
```

Comment:
Why? This very specifically was not `<=` else you can get out-of-order events when you have multiple events with the same topological token but different stream positions, hence the line below.